WebApp/static/core/js/views/desktop/TaskBar.js

```diff
@@ -185,35 +325,9 @@ Ext.define('Ext.dirac.views.desktop.TaskBar', {
 			/*
 			 * If the user is not registered
 			 */
-			if (location.protocol === 'http:') {
```

Comment:
You must keep the backward compatibility. We always have user which are not registered and does not use OAuth2